lib/amazon-pricing/version.rb

```diff
@@ -3,12 +3,12 @@
 #
 # Ruby Gem Name::  amazon-pricing
 # Author::    Joe Kinsella (mailto:joe.kinsella@gmail.com)
-# Copyright:: Copyright (c) 2011-2018 CloudHealth
+# Copyright:: Copyright (c) 2011-2019 CloudHealth
 # License::   Distributes under the same terms as Ruby
 # Home::      http://github.com/CloudHealth/amazon-pricing
 #++
 module AwsPricing
 
-VERSION = '0.1.135' # [major,minor.fix]: Moving instance-type methods to be class methods rather than instance methods
+VERSION = '2.1.000' # [major,minor.fix]: MAJOR: Gem is no longer going to be actively supported or maintained. Information in readme; reachout to cht-aws-pricing-services@groups.vmware.com for more information.
```

Comment:
the main issue when I attempted bumping (only the minor) was that all the other CHT repos 'broke' (i.e. overly restrictive gem version in Gemfile);  hence, we should test the blast radius on bumping this.